### DISCUSSION TITLE
Welcome to BigBridge Product Import Discussions!

### DISCUSSION THREAD

**Discussion by patrick-bigbridge** [bigbridge-nl/product-import#22] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Tell us about your experience with both the library, the xml import and the tools.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by Lain818** [comment]
hey hey,
any chance of sharing an example on a product delete script which deletes the product from a CSV file and SKUs?

Thanks :)

**Reply by garfix** [reply]
Hi @Lain818 

In #27 you will find an example of processing a CSV file.

If you combine this with [the information on how to delete products from their sku,](https://github.com/bigbridge-nl/product-import/blob/master/doc/importer.md#delete-products) you should be good to go :)

Let me know if you have any more questions.

**Reply by Lain818** [reply]
hey @garfix ,
will try to fiddle a bit more. i found both of the mentioned info, but it goes a bit over my head.
do i have to do it like this?
use BigBridge\ProductImport\Api\ProductDeleter;
  protected $productDeleter;

    public function __construct(
        ProductDeleter $productDeleter
    )
    {
        $this->productDeleter = $productDeleter;
    }

**Reply by garfix** [reply]
Yes, that's it

**Reply by Lain818** [reply]
@garfix Thank you very much, I succeeded. :)
I do have another question, should the bellow code work out of the box?
$global->setCountryOfManufacture('Hungaria');
I tried using it, even with a few corrections to Hungary or China or CN, nothing seems to work. Any suggestions on what I might be doing wrong?

**Reply by patrick-bigbridge** [reply]
The two-letter uppercase code should work, i.e. CN

**Reply by Lain818** [reply]
right!

**Comment by artmouse** [comment]
Hello!
Does the project support the latest versions of m2?

**Reply by garfix** [reply]
Yes, it supports up to 2.4.*

**Comment by candicehorie** [comment]
Hello, 
I am new to Magento and learning about coding, now I can import the products using XML, it is a really useful tool! it is fast and works perfectly!
But I have a few beginners questions...
To use the other ways to import the products I need to create a custom extension to get the data to import? 
and to import products using the XML API, there is any guide or what I should start research and study on how works?

**Reply by garfix** [reply]
Hi Candice,

I developed this library for BigBridge, but I left the company recently. My former colleagues will continue to maintain it. But since you have only a simple question I can answer it:

Yes to use the library you need to create an extension. You can find an example here: https://github.com/bigbridge-nl/product-import/issues/27

To use the XML API you can find examples in the doc/example directory: https://github.com/bigbridge-nl/product-import/tree/master/doc/example The library is primary; the XML API is (just) an application of it. So the full documentation was only done for the library, not for the XML.

**Comment by maupro010** [comment]
hello bro. I'm new. I want to import products from csv files. So I found your library. I have imported this to my magento project as a module. But when I ran this command: bin/magento bigbridge:product:import vendor/bigbridge/product-import/doc/example/some-products.xml. I got this error: Input file 'vendor/bigbridge/product-import/doc/example/some-products.xml' does not exist. Can you help me?
